AUTHORS.md

```diff
@@ -0,0 +1,9 @@
+Maintainers
+-----------
+Enol Fernandez <enol.fernandez@egi.eu>
```

Comment:
am I a maintainer?

Reply:
I don't wan to be alone, and you are currently reviewing a PR :)

Reply:
let's keep it them